### PR TITLE
Fix case where query isn't present for DynamicSerializerMethodField

### LINKED ISSUE #76: DynamicSerializerMethodField no parent nested_fields when no query passed in.
When `query` is not passed in, but the `DynamicSerializerMethodField` is  used, the `self.parent.nested_fields` fails because `nested_fields` wasn't set.

When `query` is passed in, it works.

### PR DESCRIPTION
**Pull request by tredzko**
Closes #76

### CODE REVIEW

File: django_restql/fields.py [hunk 1]
```diff
@@ -29,15 +30,19 @@ def to_representation(self, value):
...
-                                     **kwargs):
+
+def BaseNestedFieldSerializerFactory(
+    *args,
+    accept_pk=False,
+    create_ops=[ADD, CREATE],
+    update_ops=[ADD, CREATE, REMOVE, UPDATE],
+    serializer_class=None,
+    **kwargs
+):
```
Comment (yezyilomo) [inline django_restql/fields.py]: > Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.

https://www.python.org/dev/peps/pep-0008/#id17

Comment (tredzko) [reply django_restql/fields.py]: Ah ok, so I didn't indent this enough times, right? I'll add another indent.

**Comment by jheld** [comment]
@yezyilomo hey, thank you! can you make a 0.6.1 release?

**Comment by yezyilomo** [comment]
Actually it's going to be v0.7.0 because this release has a new feature from #63, but we have to review #78 first  so that we can merge these improvements and then release, am sure it won't take long so stay tuned.

**Pull request merged**